Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/README.md

```diff
@@ -0,0 +1,268 @@
+SaaS Security is an integrated CASB (Cloud Access Security Broker) solution that helps Security teams like yours meet 
```

Comment:
lets break this into a list

Reply:
solution that helps Security teams like yours meet the challenges of: 
-  protecting the growing availability of sanctioned and unsanctioned SaaS applications
- maintaining compliance consistently in the cloud
- stopping threats to sensitive information, users, and resources